namer/core/src/main/scala/io/buoyant/namer/ConstTransformer.scala

```diff
@@ -0,0 +1,14 @@
+package io.buoyant.namer
+
+import com.twitter.finagle.Name.Bound
+import com.twitter.finagle.{NameTree, Namer, Path}
+import com.twitter.util.Activity
+
+/** Bind the given path and use that instead of the tree. */
+class ConstTransformer(path: Path) extends NameTreeTransformer {
+
+  private[this] val bound = Namer.global.bind(NameTree.Leaf(path))
```

Comment:
also, re: use of Namer.global -- does this mean that configured namers can't be used?  should the transformer be configured with a `Path => Activity[NameTree[Name]` or something?